include/dxc/DXIL/DxilConstants.h

```diff
@@ -163,23 +163,31 @@ const unsigned kDxilMaxOloadDims = 2;
 enum class ComponentType : uint32_t {
   Invalid = 0,
   I1,
-  I16,
-  U16,
-  I32,
-  U32,
+  I16, // = 2
+  U16, // = 3
+  I32, // = 4
+  U32, // = 5
```

Comment:
Why don't we just explicitly assign all the values in this enum to make it clear that it must remain stable.

Reply:
Make those numbers explicit. Resolved.